tests/admin-unlock/main.yml

```diff
@@ -0,0 +1,316 @@
+      tags:
+        - second_install
+
+    - role: package_verify_present
+      tags:
+        - second_verify_present
+
+    - role: reboot
+
+    - role: overlayfs_verify_missing
+
+    - role: package_verify_missing
+      tags:
+        - second_verify_missing
+
+- name: Ostree Admin Unlock - Hotfix - Add/Remove Package Persistence
+  hosts: all
+  become: yes
+
+  tags:
+    - hotfix_add_remove_package_persistence
+
+  vars_files:
+    - vars.yml
+
+  pre_tasks:
+    - name: set rpm_name
+      set_fact:
+        rpm_name: "{{ g_rpm_name }}"
+
+  roles:
+    - role: ostree_admin_unlock_hotfix
+
+    - role: overlayfs_verify_present
+
+    - role: rpm_install
+
+    - role: package_verify_present
+
+    - role: rpm_uninstall
+
+    - role: package_verify_missing
+
+    - role: rpm_install
+      tags:
+        - second_install
+
+    - role: package_verify_present
+      tags:
+        - second_verify_present
+
+    - role: reboot
+
+    - role: overlayfs_verify_present
+
+    - role: package_verify_present
+      tags:
+        - third_verify_present
+
+    - role: rpm_install
+      rpm_name: "{{ g_rpm_name_2 }}"
+      tags:
+        - third_install
+
+    - role: package_verify_present
+      rpm_name: "{{ g_rpm_name_2 }}"
+      tags:
+        - fourth_verify_present
+
+- name: Ostree Admin Unlock - Hotfix - Rollback
+  hosts: all
+  become: yes
+
+  tags:
+    - hotfix_rollback
+
+  vars_files:
+    - vars.yml
+
+  pre_tasks:
+    - name: set rpm_name
+      set_fact:
+        rpm_name: "{{ g_rpm_name }}"
+
+  roles:
+    - role: rpm_ostree_rollback
+
+    - role: reboot
+
+    - role: overlayfs_verify_missing
+
+    - role: package_verify_missing
+
+    - role: package_verify_missing
+      rpm_name: "{{ g_rpm_name_2 }}"
+      tags:
+        - second_package_verify_missing
+
+- name: Ostree Admin Unlock - No Install Without Unlock
+  hosts: all
+  become: yes
+
+  tags:
+    - cannot_install_without_unlock
+
+  vars_files:
+    - vars.yml
+
+  tasks:
+    - name: set rpm_name
+      set_fact:
+        rpm_name: "{{ g_rpm_name }}"
+
+    - name: Get one of the rpms that was downloaded
+      shell: ls /root | grep {{ rpm_name }}*
+      register: rpm
+
+    - name: Attempt to install rpm
+      command: rpm -i /root/{{ rpm.stdout }}
+      register: attempted_install
+      failed_when: attempted_install.rc != 1
+
+    - include: 'roles/rpm_ostree_status/tasks/main.yml'
+
+    - name: Fail if current deployment has unlocked not set to none
+      fail:
+        when: "deployment unlocked not set to none"
+      when: "'none' not in ros_booted['unlocked']"
+
+- name: Ostree Admin Unlock - Unlock Twice - Error Message
+  hosts: all
+  become: yes
+
+  tags:
+    - unlock_twice_error_message
+
+  vars_files:
+    - vars.yml
+
+  tasks:
+    - name: Unlock system
+      command: ostree admin unlock
+
+    - include: 'roles/rpm_ostree_status/tasks/main.yml'
+
+    - name: Verify rpm_ostree status has unlocked set to hotfix
```

Comment:
I changed it to "verify rpm_ostree status is unlocked and set to hotfix".  I think that should clear things up as it can be in three states--unlocked: hotfix, unlocked: development, and unlocked: none.